src/components/override-sdk/template/DefaultForm/DefaultForm.tsx

```diff
@@ -50,12 +48,13 @@ export default function DefaultForm(props) {
 
   function settingTargetForAnchorTag() {
     const instructionDiv = document.getElementById('instructions');
-    const keyText = t('OPENS_IN_NEW_TAB');
+    const keyText = /\(([A-Za-z]+( [A-Za-z]+)+)\)/;
```

Comment:
Can you please mention what are the stings you have tested with this regex

Reply:
These are the strings I have tested - (opens in a new tab), (opens in new tab),   (yn agor mewn tab newydd) , (yn agor tab newydd)

Reply:
As discussed, please rework on this.

Reply:
I've reset this so it looks for only specific text.

Reply:
Added the text in an array and am testing each entry.